src/articulation.ts

```diff
@@ -65,7 +65,7 @@ function snapLineToStaff(canSitBetweenLines: boolean, line: number, position: nu
 // Helper function for checking if a Note object is either a StaveNote or a GraceNote.
 const isStaveOrGraceNote = (note: Note) => isStaveNote(note) || isGraceNote(note);
 
-function getTopY(note: Note, textLine: number): number {
+export function getTopY(note: Note, textLine: number): number {
```

Comment:
OK, if these three exported functions are useful for both articulation and ornaments, maybe there should be a separate utility class to store them? We can figure this out in a separate PR.